src/main/java/org/gridsuite/study/server/repository/ShortCircuitParametersEntity.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.repository;
+
+import com.powsybl.shortcircuit.StudyType;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "shortCircuitParameters")
+public class ShortCircuitParametersEntity {
+
+    public ShortCircuitParametersEntity(boolean withLimitViolations, boolean withVoltageMap, boolean withFeederResult, StudyType studyType, double minVoltageDropProportionalThreshold) {
+        this(null, withLimitViolations, withVoltageMap, withFeederResult, studyType, minVoltageDropProportionalThreshold);
+    }
+
+    @Id
+    @GeneratedValue(strategy  =  GenerationType.AUTO)
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "withLimitViolations", columnDefinition = "boolean default true")
```

Comment:
Why do you repeat the name of the column ?

Reply:
no reason in particular, just to be consistent with all the places we used this annotation